### PR TITLE
 Add initial testNg execution support

### PR DESCRIPTION
**Pull request by VijithaEkanayake**
## Purpose
$subject.

### CODE REVIEW

File: product-scenarios/http/pom.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Manuri) [inline product-scenarios/http/pom.xml]: Add license

File: product-scenarios/http/pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            <groupId>org.ballerinalang</groupId>
+            <artifactId>ballerina-scenario-test-commons</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.6</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (Manuri) [inline product-scenarios/http/pom.xml]: Shall we move this dependency to the dependencyManagement section of root pom and reuse it here?

File: product-scenarios/http/spotbugs-exclude.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<FindBugsFilter>
+    <!-- These bugs are from auto generated code of protobuff plugin-->
+    <Match>
+        <Package name="org.ballerinalang.test.util.grpc"/>
+    </Match>
+</FindBugsFilter>
```
Comment (Manuri) [inline product-scenarios/http/spotbugs-exclude.xml]: Shall we add a newline

File: product-scenarios/http/src/test/resources/logging.properties [hunk 4]
```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Manuri) [inline product-scenarios/http/src/test/resources/logging.properties]: Need to update year

File: product-scenarios/scenarios-commons/pom.xml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Manuri) [inline product-scenarios/scenarios-commons/pom.xml]: Update year

File: product-scenarios/pom.xml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Manuri) [inline product-scenarios/pom.xml]: Add license

File: product-scenarios/spotbugs-exclude.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<FindBugsFilter>
+    <!-- These bugs are from auto generated code of protobuff plugin-->
+    <Match>
+        <Package name="org.ballerinalang.test.util.grpc"/>
+    </Match>
+</FindBugsFilter>
```
Comment (Manuri) [inline product-scenarios/spotbugs-exclude.xml]: Add newline

**Comment by VijithaEkanayake** [comment]
Fixes: https://github.com/ballerina-platform/ballerina-lang/issues/14192

**Pull request merged**